src/@next/Card/CardStyle.ts

```diff
@@ -15,8 +15,8 @@ export const StyledCardContainer = styled.div`
 export const StyledCardHeaderWrapper = styled.div`
   display: flex;
   padding: ${space24} ${space24} 20px;
-  flex-wrap: wrap;
-  flex-direction: column;
+  flex-wrap: nowrap;
+  flex-direction: row;
```

Comment:
tbh I am concerning about this... if someone put a few div as the header content (custom header), the default behavior changes, the div will be in row instead of column